src/bundle/Resources/views/admin/section/list.html.twig

```diff
@@ -26,20 +26,21 @@
         <div class="ez-table-header">
             <div class="ez-table-header__headline">{{ 'section.list.title'|trans|desc('Sections') }}</div>
             <div>
-                <a title="{{ 'section.new'|trans|desc('Create a new Section') }}"
-                   href="{{ path('ezplatform.section.create') }}"
-                   class="btn btn-primary" {% if not can_edit %} data-disabled{% endif %}>
-                    <svg class="ez-icon ez-icon-create">
-                        <use xmlns:xlink="http://www.w3.org/1999/xlink"
-                             xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#create"></use>
-                    </svg>
-                </a>
+                {% if can_edit %}
+                    <a
```

Comment:
Why we changing code style? So far (and in this PR) we used `                <a title="{{ 'section.new'|trans|desc('Create a new Section') }}"
                   href="{{ path('ezplatform.section.create') }}"
                   class="btn btn-primary" {% if not can_edit %} data-disabled{% endif %}>`

Reply:
@mikadamczyk  It's more readable for frontend devs.